bootstrap/config/kfctl_gcp_iap.0.6.yaml

```diff
@@ -62,6 +62,9 @@ spec:
     katib-ui:
     - name: overlay
       value: istio
+    metadata:
+    - name: overlay
+      value: application
```

Comment:
I thought you need overlay istio?

Reply:
Here is the manifest dir: https://github.com/kubeflow/manifests/tree/master/metadata/overlays/istio

Should this value be `istio`?

Reply:
QQ: I don' need to change any file with comments like this, right?
```
# TODO(jlewi): This file is legacy config for bootstrapper.
# we can delete it once bootstrapper is using kfctl and KfDef.
......
```